src/frontends/tensorflow/src/input_model.cpp

```diff
@@ -824,6 +824,14 @@ ov::frontend::Place::Ptr InputModel::get_place_by_tensor_name(const std::string&
     return _impl->get_place_by_tensor_name(tensorName);
 }
 
+ov::frontend::Place::Ptr InputModel::get_place_by_input_index(size_t input_idx) const {
+    const auto& inputs = get_inputs();
+    if (input_idx < inputs.size()) {
+        return inputs.at(input_idx);
+    }
+    return {};
```

Comment:
we do not need it. Let us throw exception that is not implemented

Reply:
This implementation mimics the previous behavior, lets have it here so we can use the same code to process places in future

Reply:
For me, it is not safe change. We need to run full validation.
I am leaning to revert it for TF FE because I need to sure that it is not used for TF FE as now

Reply:
ok, removed